Prepare for release

kgreav opened the pull request:
Signed-off-by: Kurt Greaves <kurt.greaves@zepben.com>

ryanjcoleman commented:
Task linked: [PROJ-1338 Publish pp-translator to pypi](https://app.clickup.com/t/6929263/PROJ-1338)

kgreav commented:
So for reference, i've just fixed up licencing, added some metadata fields to the setup.py (not all should be required, but it's best practice). Also made it depend on latest released version of the sdk (0.25.0) rather than prerelease (not necessary but cleaner). Other than that not much needed to be done to make the build work.

Added tox for testing build (which is used by the release workflow) and some coverage config (note had to drop coverage to 10% for it to pass)
Had to add the docs folder + changelog and what not as the builds rely on them existing.

I've copied the workflows from the python sdk, and after merging this I'll have have to test/make work the snapshot and release builds.

we may want to consider changing the package  to be in the zepben namespace like the sdk and eas client, but that can be done later.
